provider/kubeletconfig/kubeletconfig_validators.go

```diff
@@ -50,6 +50,10 @@ func (p PidsLimitValidator) MarkdownDescription(_ context.Context) string {
 
 func (p PidsLimitValidator) ValidateInt64(_ context.Context, req validator.Int64Request, resp *validator.Int64Response) {
 
+	if req.ConfigValue.IsNull() || req.ConfigValue.IsUnknown() {
```

Comment:
Please use the common function here
something like:
```suggestion
	if !common.HasValue(req.ConfigValue) {
```